src/test/java/org/shikshalokam/backend/scp/SelfCreationPortalCreateNewPermission.java

```diff
@@ -62,10 +61,95 @@ public void testCreateNewPermission() {
                 .body(requestBody)
                 .when().post(createPermissionEndpoint);
 
-        // Logging the response body
-        logger.info("Response Body: " + response.getBody().asString());
-        logger.info("Response Status Code: " + response.getStatusCode());
+        // Log the status code and response body
+        int statusCode = response.getStatusCode();
+        String responseBody = response.getBody().asString();
+        logger.info("Response Status Code: " + statusCode);
+        logger.info("Response Body: " + responseBody);
+        // Pretty print the response to the console or logs
+        response.prettyPrint();
+
+        //log the end call for createPermission API
+        logger.info("Ended calling the CreatePermission API with valid payload.");
```

Comment:
Please add 
assertEquals to validate response code .
fetch responce and validate ID 
No assertions